wiki/Glossary/en.md

```diff
@@ -619,7 +670,8 @@ In contrast, textures are scaled such that one texture pixel ([texel](https://en
 Ouendan has three meanings:
 
 - In Japanese culture, a cheer squad which, similar to western cheerleaders, rallies a sports team on using drums, horns, banners and yells of encouragement.
-- The pair of Japanese DS games that osu! was based on. This term may sometimes be used to refer specifically to the first of these games.
+- The pair of Japanese DS games that osu! was based on.
+  This term may some times be used to refer specifically to the first of these games.
```

Comment:
`  This term may some times be used to refer specifically to the first of these games.`